### DISCUSSION TITLE
backup recommandation

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy-znp#151] [General]
What are the recommendations / experiences in terms of backup/restore

Is nvram backup the best comparing to the network backup ?
Is the nvram backup/restore only possible when on the same board/brand or if we stay on the same board we can use it ?

**Comment by puddly** [comment]
I've outlined the differences between the two in the documentation, including under what conditions you can restore the NVRAM backup to another device: https://github.com/zigpy/zigpy-znp/blob/dev/TOOLS.md#backup-and-restore

I would suggest using the network backup. The format will change in the future.

**Reply by pipiche38** [reply]
thanks this is really great
